### DISCUSSION TITLE
Idea: Create a whatsapp.txt list separated from the list facebook.txt

### DISCUSSION THREAD

**Discussion by CYB3RL4MBD4** [blocklistproject/Lists#239] [Ideas]
Hi, first of all, thank you so much for all the good work!

The new WhatsApp (non-)privacy policy that will come soon, makes a lot of people move to another messaging app but they would like to continue to use Facebook.

I know that WhasApp belongs to Facebook but would it be an idea to create a new list for WhatsApp (whatsapp.txt) domains but separated from the facebook.txt list?

So people could block WhatsApp but not necessarily Facebook.

Many thanks for your time and have a great day.

**Comment by CYB3RL4MBD4** [comment]
Oh, my bad, I first put this idea to the wrong topic, sorry. Many thanks for correcting.

**Reply by blocklistproject** [reply]
I'll see if I can get this done today for you. It may not be a published list unless other people ask for this as well.

**Reply by CYB3RL4MBD4** [reply]
Many thanks 😀

**Comment by CYB3RL4MBD4** [comment]
Anybody else interested in a separate WhatsApp list? For people who want to block WhatsApp but not necessarily Facebook. Thanks for voting 😉

**Reply by blocklistproject** [reply]
I'll get it done for you today!

**Reply by blocklistproject** [reply]
This is done. It is under the beta list section for now until we can get some feedback

**Reply by CYB3RL4MBD4** [reply]
> This is done. It is under the beta list section for now until we can get some feedback

Many thanks! 😀
